lib/core/src/stringOpr.cpp

```diff
@@ -316,7 +316,7 @@ checkStringForSystem( const char * inString ) {
     if ( inString == NULL ) {
         return 0;
     }
-    if ( boost::regex_match( inString, boost::regex( "[a-zA-Z0-9,./ ]*" ) ) ) {
+    if (boost::regex_match(inString, boost::regex("[^\r\n]*"))) {
```

Comment:
let's get a comment to say in words what this regex is doing (and why)... to be kind for the next reader/debugger.

Reply:
Now that we're using boost process, is this check still necessary?

Reply:
Ah, so this check is actually for spec reasons: subject lines aren't allowed to include \r or \n by spec...thing is, in my tests, `mail` handles it just fine (by simply not adding a subject line)...but I thought it'd be good to still check.

Reply:
yep - leave it and a comment saying the 'why'.